content/docs/plugins/notifications-plugin/_index.md

```diff
@@ -20,9 +25,40 @@ Whenever a new notification is detected the frontend pops an alert to the user.
 # Backend
 The backend plugin provides the backend application for reading and writing notifications.
 
+## Authentication
+
+Two kinds of authentation are in place with Notifications Plugin:
+
+1) For reading notifications the token generated by backstage IDP with typ `vnd.backstage.user` is used. The JWT claim `sub` will be the one used to filter the notifications.
```

Comment:
just to make things a bit clearer
s/the token/the JWT token

Can you add a link for that in case this is documented nicely on backstage side.

Reply:
Sadly, this was digging into a rabbit hole in the code, there is no doc in backstage at all, and things can be changed using the customResolvers, etc..

https://backstage.io/docs/auth/identity-resolver